vcd/datasource_vcd_nsxt_network_imported.go

```diff
@@ -81,6 +81,35 @@ func datasourceVcdNsxtNetworkImported() *schema.Resource {
 				Computed:    true,
 				Description: "Network prefix",
 			},
+			"static_ip_pool": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "IP ranges used for static pool allocation in the network",
+				Elem:        networkV2IpRangeComputed,
+			},
+			"dual_stack_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Boolean value if Dual-Stack mode is enabled",
+			},
+			"secondary_gateway": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Computed:    true,
+				Description: "Secondary gateway (can only be IPv6 with dual_stack_enabled=true)",
```

Comment:
Maybe we should code-quote this everywhere?

```suggestion
				Description: "Secondary gateway (can only be IPv6 with `dual_stack_enabled=true`)",
```

Though I'm not sure would that be a consistent approach in a wider sense.

Reply:
I can change this to avoid code. Something like:
> Secondary gateway (can only be IPv6 and requires enabled Dual Stack mode)

Reply:
I personally liked the code approach, but the soft wording approach may be better here, as it requires one to research dual stack mode before actually enabling it.

Reply:
Changed it in this place and all other places to
> XXX (can only be IPv6 and requires enabled Dual Stack mode)